pkg/proc/native/proc_darwin.go

```diff
@@ -422,7 +422,8 @@ func (dbp *nativeProcess) resume() error {
 }
 
 // stop stops all running threads and sets breakpoints
-func (dbp *nativeProcess) stop(trapthread *nativeThread) (err error) {
+func (dbp *nativeProcess) stop(ptrapthread **nativeThread) (err error) {
```

Comment:
I'm not all that excited about the pointer-to-pointer change.

I think from a clarity of intent perspective it's cleaner to keep it as it, but instead return `(*nativeThread, err)` and let the caller do something like `trapthread = dbp.stop(trapthread)`.

Thoughts?

Reply:
Done.